consul/operator_endpoint.go

```diff
@@ -183,3 +184,63 @@ func (op *Operator) AutopilotSetConfiguration(args *structs.AutopilotSetConfigRe
 	}
 	return nil
 }
+
+// Used by Autopilot to query the raft stats of the local server.
+func (op *Operator) RaftStats(args struct{}, reply *map[string]string) error {
+	*reply = op.srv.raft.Stats()
```

Comment:
Let's make a struct here and copy just the bits we care about into something on the Consul side. The -stage-two version of the Raft library introduces a structure vs. the map interface. This also helps to make sure this is as minimal as possible so we don't leak any info, since this doesn't have ACL control.